app/src/main/java/org/xbmc/kore/ui/sections/remote/PlaylistFragment.java

```diff
@@ -170,10 +214,9 @@ public void onDestroyView() {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.action_clear_playlist:
-                Playlist.Clear action = new Playlist.Clear(currentPlaylistId);
+                int playlistId = playlists.get(playlistsBar.getSelectedPlaylistType()).getPlaylistId();
+                Playlist.Clear action = new Playlist.Clear(playlistId);
                 action.execute(hostManager.getConnection(), defaultStringActionCallback, callbackHandler);
-                // If we are playing something, refresh playlist
-                forceRefreshPlaylist();
```

Comment:
Sure we don't need to refresh the playlists after a clear?

Reply:
Yes, it's now performed when registering the playlist observer in onResume

Reply:
This is the code for the menu clear playlist function, i fail to see the connection to `onResume`.

Reply:
Ah my bad. As we now handle the onClear notification it is not needed here anymore. Although I do still need to fix handling HTTP only connections. I'm thinking of just calling `playlistOnClear` here and not wait for the HTTP checker to finally call the `playlistOnClear`.